source/maze/MouseController.cpp

```diff
@@ -186,7 +186,7 @@ void MouseController::updateCell()
 {
 	Position tmp = getCurrentPos();
 	Maze::updateCell();
-	Maze::setMouse(tmp.row, tmp.col);
+	mazeIO.setMousePosition(tmp);
```

Comment:
Can you delete this method `updateCell()` and use this line by overriding `Maze::printCell()` in `MouseController` class?